src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -25,5 +25,25 @@
 
 @RunWith(JUnit4.class)
 public final class UtilityTest {
-  
+  @Test
+  public void convertUsingGsonToJsonTest() {
+    // Object with attributes.
+    SubjectTag tag = new SubjectTag(5, "Interviews", "red");
+    String expectedJson = "{\"id\":5,\"subject\":\"Interviews\",\"color\":\"red\"}";
+    
+    String actual = Utility.convertToJsonUsingGson(tag);
+    
+    Assert.assertEquals(expected, actual);
```

Comment:
Make sure you run your tests every time you make changes! 'expected' is no longer defined, so this will fail :)

Also, for consistency's sake, if you're writing 'expectedJson', also write 'actualJson'.

Same with the test below!

Reply:
I am so sorry, had my mind elsewhere! Done